.github/workflows/main.yml

```diff
@@ -20,7 +20,22 @@ jobs:
 
       - name: Build
         run: go build ./...
+  nextjs-build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ui
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
 
+      - name: Build frontend
+        run: |
+          bun install
```

Comment:
What si this `bun` thing? Do we really need it? Why add this dependency in addition to next?

Reply:
It is a faster alternative to npm. We can use that too, but npm's action was complaining about some random dependency issue that I couldn't resolve. npm, bun, deno and a bunch of others are all javascript runtimes but all use the same v8 engine in chrome for javascript, so they produce the same javascript code but their tooling around transpiling is a bit different.

Reply:
It is not a dependency in gocert, neither the backend nor the frontend, it is only in the workflow, and installs the package.json and runs next build just like node would.

Reply:
I'm not confident about adding this dependency to fix another issue. Let's try to fix the issue. Plus we're not even using `bun` when running building or running the service in the rock.

Reply:
gone